bench/db_bench.ml

```diff
@@ -165,11 +184,11 @@ module Lmdb = struct
   let fail_on_error f =
     match f () with Ok _ -> () | Error err -> failwith (string_of_error err)
 
-  let flags = [ Lmdb.NoRdAhead; Lmdb.NoSync ]
+  let flags = [ Lmdb.NoRdAhead; Lmdb.NoSync; Lmdb.NoSync ]
```

Comment:
duplicated `NoSync`?
for the sequential reads tests it might make sense to remove the `NoRdAhead` flag as it will help the OS to pre-fetch sequential pages.

Reply:
> duplicated `NoSync`?

typo, it was supposed to be ` Lmdb.NoMetaSync` 

> for the sequential reads tests it might make sense to remove the `NoRdAhead` flag as it will help the OS to pre-fetch sequential pages.